src/efi.rs

```diff
@@ -22,27 +23,64 @@ use crate::ostreeutil;
 use crate::util;
 use crate::util::CommandRunExt;
 
-/// The path to the ESP mount
-pub(crate) const MOUNT_PATH: &str = "boot/efi";
+/// The ESP partition label
+pub(crate) const ESP_PART_LABEL: &str = "EFI-SYSTEM";
+
+#[macro_use]
+lazy_static! {
+    /// The path to a temporary ESP mount
+    static ref MOUNT_PATH: PathBuf = {
+        // Create new directory in /tmp with randomly generated name at runtime for ESP mount path.
+        tempfile::tempdir_in("/tmp").expect("Failed to create temp dir for EFI mount").into_path()
```

Comment:
Any particular reason we're not using /run that is root:root owned instead?

Reply:
Not really any particular reason. Just read up on `/tmp` vs `/run`, I'm guessing `/run` would be more appropriate?

Reply:
Or /run/boot_efi/$VOLUME-ID to account for the multiple ESP case?

Reply:
This PR currently doesn't support the multiple ESP case and will output an error if there's no single device with the label `EFI-SYSTEM`, but I guess it'll still be clearer to indicate which ESP even if we only have one?